examples/memeolist.variables.json

```diff
@@ -0,0 +1,9 @@
+{ 
+  "ownerid": "1", 
+  "photourl": "https://i.imgur.com/W4dhL0b.jpg", 
+  "owner": "mmartini", 
+  "email":"mmartin@rodhut.com", 
+  "displayname": "Martin Martinski", 
+  "pictureurl": "https://tinyurl.com/avataraerogear",
```

Comment:
May, the variables could not be `pictureUrl` instead of `pictureurl` and `displayName` instead of `displayname`? What do you think?

Reply:
I actually had the initial schema with camel case. But field names are converted to lowercase because you need to wrap column names in quotes for Postgres case-sensitivity. @wtrocki is it a lot of work to fix the casing and making everything camel case? Any side effects on client side?